presto-iceberg/pom.xml

```diff
@@ -681,7 +677,14 @@
                             <dependency>com.facebook.airlift:node:jar</dependency>
                             <dependency>javax.servlet:javax.servlet-api:jar</dependency>
                             <dependency>org.apache.httpcomponents.core5:httpcore5:jar</dependency>
+                            <dependency>software.amazon.awssdk:regions:jar</dependency>
+                            <dependency>software.amazon.awssdk:sdk-core:jar</dependency>
+                            <dependency>software.amazon.awssdk:auth:jar</dependency>
+                            <dependency>software.amazon.awssdk:aws-core:jar</dependency>
```

Comment:
Is there any reason for doing this? Since we are using these dependencies, we should include these dependencies directly rather than adding the exceptions here.